projects/helper/balancer.js

```diff
@@ -1,7 +1,7 @@
 const { sumTokens2 } = require('./unwrapLPs');
 const { getLogs } = require('./cache/getLogs')
 
-function onChainTvl(vault, fromBlock, { blacklistedTokens = [], preLogTokens = [], onlyUseExistingCache } = {}) {
+function onChainTvl(vault, fromBlock, permitFailure, { blacklistedTokens = [], preLogTokens = [], onlyUseExistingCache } = {}) {
```

Comment:
can you add permitFailure as one of the optional options instead of new parameter